docs/06-cicd.md

```diff
@@ -0,0 +1,68 @@
+# Continuous Integration and Delivery
+## Content
+* [Deliver a new release](#triggering-a-delivery-using-the-ci)
+* [Workflows](#workflows)
+* [Determining the next version](#determining-the-next-version)
+* [Used services](#services)
+* [Hints and quirks](#hints-and-quirks)
+
+## Triggering a Delivery using the CI
+
+The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflows
+(*api_triggered_development_delivery* and *api_triggered_production_delivery* depending on the `production_delivery` parameter):
+
+* Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-pi-tokens/).
+* Trigger a build using the tool [trigger-pipeline.sh](../.circleci/trigger-pipeline).
+* For more information on how to use it, execute it without parameters to see usage information.
+
+## Workflows
+
+See the table below for a detailed overview of all existing workflows:
+
+|Workflow|Trigger|Checks|Builds|Bump version|Delivery|
```

Comment:
Either Bumb version or version bumb. Decide, but stay consistent (see line 35)

Reply:
Fixed in line 35.